api-reference/50 Common/utils/pdfExporter/exportGantt(options).md

```diff
@@ -24,15 +24,14 @@ This method requires the <a href="https://github.com/MrRio/jsPDF" target="_blank
 
 The **exportGantt(options)** method allows you to save information about the Gantt chart's layout, appearance, and tasks. This method supports the following options:
 
-- **createDocumentMethod** - Specifies a function that creates a PDF document.
-- **format** {string | width?: number, height?: number } - Specifies the document size.
-- **landscape** {boolean} - Specifies whether to use horizontal orientation for the document.
-- **fileName** {string} - Specifies the file name.
-- **exportMode** {"all", "treeList", "chart"} - Specifies the part of the component to export (chart area, tree list area, or the entire component).
-- **dateRange** {"all" | "visible" | startDate? : Date, endDate? : Date, startIndex? : number, endIndex? : number } - Restricts data output against a specified date range.
-- **margins** { left?: number, top?: number, right?: number, bottom?: number } - Specifies the outer indents of the exported area.
-
-The exporter supports standard PDF fonts. Refer to the [Use of Unicode Characters / UTF-8](https://github.com/parallax/jsPDF#use-of-unicode-characters--utf-8) article to get information on how to use custom fonts for exporting.
+- [createDocumentMethod](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#createDocumentMethod) - Specifies a function that creates a PDF document.
+- [format](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#format) - Specifies the document size.
+- [landscape](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#landscape) - Specifies whether to use horizontal orientation for the document.
+- [fileName](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#fileName) - Specifies the file name.
+- [exportMode](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#exportMode) - Specifies the part of the component to export (chart area, tree list area, or the entire component).
+- [font](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/font/) - Specifies the font.
+- [dateRange](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#dateRange) - Restricts data output against a specified date range.
+- [margins](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#margins) - Specifies the outer indents of the exported area.
```

Comment:
You can simply give a link to the PdfExportGanttProps docs. This way you wouldn't have to update this list if PdfExportGanttProps will be added another prop.